README.md

```diff
@@ -9,6 +9,9 @@ files (``*.proto``) in your project. There are two pieces of its job:
  2. It adds the generated Java source files to the input of the corresponding
     Java compilation unit (_sourceSet_ in a Java project; _variant_ in an
     Android project), so that they can be compiled along with your Java sources.
+    - Note if you are generating non-Java/Kotlin source files, they will not be
+    included for compilation automatically, you will need to add them to sources
+    for language-specific compilations. See details in [this section](#default-outputs).
```

Comment:
Nit: It's nice to avoid "this" and "here" links. How about "See \[Default outputs](#default-outputs) for examples." Or "See details in the \[Default options section](#default-outputs)." (Or something along those lines)